pkg/appgw/backendhttpsettings.go

```diff
@@ -205,8 +205,11 @@ func (c *appGwConfigBuilder) generateHTTPSettings(backendID backendIdentifier, p
 		Name: &httpSettingsName,
 		ID:   to.StringPtr(c.appGwIdentifier.HTTPSettingsID(httpSettingsName)),
 		ApplicationGatewayBackendHTTPSettingsPropertiesFormat: &n.ApplicationGatewayBackendHTTPSettingsPropertiesFormat{
-			Protocol: n.HTTP,
-			Port:     to.Int32Ptr(int32(port)),
+			Protocol:                       n.HTTP,
+			Port:                           to.Int32Ptr(int32(port)),
+			PickHostNameFromBackendAddress: to.BoolPtr(false), // setting to default
```

Comment:
different question here...
if we don't specify a field such as "PichHostName..." here, would appgw give it a default value as well?